### ISSUE TITLE
Female Orc Terrain Problem

### ISSUE THREAD

**Issue by smashy7** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3578]
The problem is that you get stuck in terrain...arena/bg/even in world...fix this...pls
http://www.wow-one.com/forum/topic/95359-orc-female-bug/

**Comment by teddys** [comment]
this is not a problem only for female orc .. i often get stuck on tomb in ruins of lordaeron or below the bridge in blades edge arena

**Comment by smashy7** [comment]
that is true...if i use  [Winterfall Firewater] i don't get stuck anymore , so i guess it's a size problem or something

**Comment by smashy7** [comment]
up fix please i get stuck while others dont...it's not fair

**Comment by Kaiitothewai** [comment]
>_>

**Comment by smashy7** [comment]
pls fix this developers !

**Comment by Kaiitothewai** [comment]
You are aware there's alot of issues on this BugTracker and yours isn't exactly a priority, I'm sure people would rather see movement fixed then this? :)

**Comment by warlee** [comment]
Issue exists not only on orc female but as a most of races/genders, there is needed a lot of more information and work on this report.
What is needed for start is every arena zone (Blade's Edge Arena, Ruins of Lorderon, Nagrand Arena) having seperated report and recorded every 'bug' spot with specific explanation which race/gender gets stuck there

**Comment by smashy7** [comment]
The tauren male,orc male, undead male has no problem in the spot where i frapsed that video...
